docs/ops/detection/ExperimentalDetectronROIFeatureExtractor_6.md

```diff
@@ -0,0 +1,138 @@
+## ExperimentalDetectronROIFeatureExtractor <a name="ExperimentalDetectronROIFeatureExtractor"></a> {#openvino_docs_ops_detection_ExperimentalDetectronROIFeatureExtractor_6}
+
+**Versioned name**: *ExperimentalDetectronROIFeatureExtractor-6*
+
+**Category**: Object detection
+
+**Short description**: *ExperimentalDetectronROIFeatureExtractor* is the [ROIAlign](ROIAlign_3.md) operation applied 
+over a feature pyramid.
+
+**Detailed description**: *ExperimentalDetectronROIFeatureExtractor* maps input ROIs to the levels of the pyramid 
+depending on the sizes of ROIs and parameters of the operation, and then extracts features via ROIAlign from 
+corresponding pyramid levels.
+
+Operation applies the *ROIAlign* algorithm to the pyramid layers:
+
+* output[i, :, :, :] = ROIAlign(inputPyramid[j], rois[i])
+* j = PyramidLevelMapper(rois[i])
```

Comment:
Lets wrap these lines with ` characters as we do for most formulas

Reply:
Done.